optimum/exporters/tasks.py

```diff
@@ -222,7 +251,6 @@ class TasksManager:
             "text-classification": "TFAutoModelForSequenceClassification",
             "token-classification": "TFAutoModelForTokenClassification",
             "multiple-choice": "TFAutoModelForMultipleChoice",
-            "object-detection": "TFAutoModelForObjectDetection",
```

Comment:
why removed?

Reply:
a lot of the tasks and their `TFAutoModel` actually don't exist for TF like:
`TFAutoModelForObjectDetection`, `TFAutoModelForAudioFrameClassification`, `TFAutoModelForAudioXVector` are nowhere to be found in [modeling_tf_auto](https://github.com/huggingface/transformers/blob/main/src/transformers/models/auto/modeling_tf_auto.py)